lua/focus/modules/resizer.lua

```diff
@@ -26,15 +26,19 @@ end
 -- TODO: Support: Vista.vim
 -- TEST: floating windows, snap/telescope, toggleterm, trees, scrollview.nvim, blank buffer, popups during autocompletion i.e coq
 function M.split_resizer(config) --> Only resize normal buffers, set qf to 10 always
+	if vim.g.enabled_focus_resizing == 0 or vim.api.nvim_win_get_option(0, 'diff') then
```

Comment:
I haven't tested this with other differing scenarios yet and may need to be `vim.api.nvim_win_get_option(0, 'diff') and ft =~ nil` clause instead 🤔